test/test.py

```diff
@@ -763,6 +764,29 @@ def test_name_substitution(self):
             with_slash_after = set(f1 for f1 in files_after if '/' in f1)
             self.assertEqual(expected_slash_after, with_slash_after)
 
+    def test_project_accessions(self):
+        uuid = 'eca05046-3dad-4e45-b86c-8720f33a5dde'
+        version = '2019-03-17T220646.332108Z'
+        manifest, metadata_files = self._canned_bundle('staging', uuid, version)
+
+        # Append an additional value here to assert that multiple Accessions
+        # with the same type property are possible.
+        metadata_files['project_0.json']['insdc_project_accessions'].append('SRP000001')
```

Comment:
In this case I would modify the can and commit b/c it enables us to extend the existing test rather than writing yet another overlapping one.